examples/2d/parameters_hyp_diff_nonperiodic.toml

```diff
@@ -5,6 +5,10 @@ equations = "HyperbolicDiffusionEquations"
 nu = 1.0
 
 initial_conditions = "initial_conditions_poisson_nonperiodic"
+boundary_conditions = ["boundary_conditions_poisson_nonperiodic",
+                       "boundary_conditions_poisson_nonperiodic",
+                       "nothing",
+                       "nothing"]
```

Comment:
`nothing` means periodic?

Reply:
Yes, or "do not apply a boundary condition here for whatever reason" (this could be the case, e.g., when we couple two trees in a surface-coupled multi-physics setting).

Reply:
Okay, thanks.